packages/server/src/fhir/operations/expand.ts

```diff
@@ -96,20 +96,17 @@ export function filterIncludedConcepts(
   system?: string
 ): ValueSetExpansionContains[] {
   const filter = params.filter?.trim().toLowerCase();
-  const codings: Coding[] = flattenConcepts(concepts, { filter, system });
-  if (!filter) {
-    return codings;
-  }
-  return codings.filter((c) => c.display?.toLowerCase().includes(filter));
```

Comment:
This filter is safe to drop because it's already being applied inside `flattenConcepts()` above. Nice clean up!